### PR TITLE
Add Wavestack release page

### PR DESCRIPTION
**Pull request by matfechner**

### CODE REVIEW

File: docs/content/en/support/releases.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Release"
```
Comment (wwentland) [inline docs/content/en/support/releases.md:2]: Should this be "Releases" ?

File: docs/content/en/support/releases.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+linkTitle: "Releases"
+description: "Information on upgrades and maintenance procedures for Wavestack services."
+type: "docs"
+weight: 1
+
+---
+
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+
+<!-- Copyright (C) 2023 Wavecon GmbH -->
```
Comment (wwentland) [inline docs/content/en/support/releases.md:12]: Please make sure that the weight is what we want it to be as the order of topics is based on it. Specifically, we already have an article with `weight: 1` and the fallback would be alphanumerical sorting.

I'd also kindly ask you to remove those spurious blank lines. Thanks!

File: docs/content/en/support/releases.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+#### Release Notes Wavestack 2023.11 - Antelope - SCS R5
+
+###### Announcement  26.10.2023
+
+From Monday 23.10.2023 to Wednesday 25.10.2023 we upgraded wavestack to SCS  Release 5.
+
+Our upgrade included OSISM 6.0.0 and OpenStack Antelope 2023.1. This upgrade shipped several OpenStack-based changes for further reliability as well as core
+ fixes and improvements as they came up on OpenDev in the last few 
+month.
```
Comment (wwentland) [inline docs/content/en/support/releases.md:22]: Nit: Might want to re-fill this paragraph for consistent line length and indentation.

Comment (linwalth) [reply docs/content/en/support/releases.md]: fixed

**Comment by linwalth** [comment]
@matfechner you need to get the current state of the PR and then sign it. else i cant merge it

File: docs/content/en/support/releases.md (hunk 2 above)
Comment (linwalth) [reply docs/content/en/support/releases.md]: fixed

**Comment by matfechner** [comment]
will  refactor and is outedated

**Pull request closed without merging**